app/assets/stylesheets/common/base/compose.scss

```diff
@@ -643,7 +643,6 @@ body:not(.ios-safari-composer-hacks) {
       min-height: calc(var(--min-height) - 4em);
     }
     padding-bottom: var(--composer-ipad-padding);
-    padding-bottom: calc(10px + env(keyboard-inset-height));
```

Comment:
Change 1: remove `env(keyboard-inset-height)`, on Android it's particularly an issue because Chrome and Firefox react very differently to it.